src/bdscript/toTitleCase.md

```diff
@@ -0,0 +1,10 @@
+# $toTitleCase
+Changes provided text to lowercase.
```

Comment:
It doesn't change the provided text to lowercase.
It transforms "this kind of text" into "This Kind Of Text".

Reply:
Okay